### DISCUSSION TITLE
Cannot start backend for development

### DISCUSSION THREAD

**Discussion by Klengen** [BuchholzTim/Whitebird#88] [Setup Help]
Hi, i wanted to use your project and add some features. I can start the MongoDB, and the front-end like described in the "Get started". However when I try to start the back-end with _yarn start:dev_ from the backend folder, I run into this exception: 
![image](https://user-images.githubusercontent.com/39967645/123406077-d2a33500-d5aa-11eb-939f-00b54499f24b.png)

As well, I tried it on two different Windows 10 computers. Maybe somebody had the same problem or knows what i am missing.

I would really appreciate some help and if more information are needed, feel free to ask.

PS: Cooles Projekt!

**Comment by MarcBollmann** [comment]
Did u start the mongo container with the provided docker-compose? Can you verify in the Docker Dashboard that the DB is running?

**Comment by Klengen** [comment]
Thanks for the fast response.

And yes, it should be running.
![image](https://user-images.githubusercontent.com/39967645/123409265-48f56680-d5ae-11eb-8829-e11e8b44bd6b.png)

**Comment by BuchholzTim** [comment]
The URI has to be formatted as a [Standard Connection String for MongoDB](https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-standard-connection-string-format). 

Assuming everything runs on the same machine you should be able to just replace `@mongodb` with `@localhost` and it should work.

`MONGO_URI=mongodb://root:asdasd@localhost`

I'm going to fix the wiki today.

**Comment by Klengen** [comment]
Oh thanks a lot. It is working :D

Never used docker before :)
